KlingonAssistant/data/mem-05-H.xml

```diff
@@ -925,7 +925,7 @@ A physical path would be a {taw:n}.</column>
       <column name="entry_name">Heghba'</column>
       <column name="part_of_speech">n:alt,fic</column>
       <column name="definition">{Heghbat:n}</column>
-      <column name="definition_de"></column>
+      <column name="definition_de">{Heghbat:n}</column>
```

Comment:
No, don't remove them now that you've entered them. It's too much work now, and it's better to be consistent. If doing it this way makes it easier for you to do a search for untranslated German definitions, that's fine.